packages/dendron-cli/src/commands/devCLICommand.ts

```diff
@@ -412,6 +417,65 @@ export class DevCLICommand extends CLICommand<CommandOpts, CommandOutput> {
     return { staticPath };
   }
 
+  syncTutorial() {
+    const dendronSiteVaultPath = path.join(
+      BuildUtils.getLernaRoot(),
+      "docs",
+      "seeds",
+      "dendron.dendron-site",
+      "vault"
+    );
+
+    const tutorialDirPath = path.join(
+      BuildUtils.getPluginRootPath(),
+      "assets",
+      "dendron-ws",
+      "tutorial"
+    );
+
+    const commonDirPath = path.join(tutorialDirPath, "common");
+
+    // wipe everything in /assets/dendron-ws/tutorial/treatments
+    const treatmentsDirPath = path.join(tutorialDirPath, "treatments");
+
+    fs.removeSync(treatmentsDirPath);
+    fs.ensureDirSync(treatmentsDirPath);
+
+    // grab everything from `tutorial.*` hierarchy
```

Comment:
nit: maybe expand on this comment to say exclude 'tutorial.md' because the websites' version says to download vs code, etc.

Reply:
that's not related to this piece of code.

what you are describing would be already treated when we are updating the `tutorial` hierarchy in `dendron-site`.

We exclude `tutorial.md` here because the actual tutorials are in `tutorial.{treatmentName}.*`.

`tutorial.md` is just there to add a warning message that this is not the hierarchy that is being published to the wiki. We don't want to sync this note because it's not actually part of the tutorial asset.